### PR TITLE
Edit homepage

### PR DESCRIPTION
**Pull request by kdweeks**
I believe I have updated the homepage to Philip's requirements and pushed everything correctly back to the 'edit_homepage' branch.   Please check these changes out for a merge to 'vt_6_x_dev'

I wasn't sure if I should add Reviewers or Assignees so I added @alawvt and @keithgee to both. 

This pull request resolves issue #652.

### CODE REVIEW

**Comment by keithgee** [comment]
@kdweeks I'll take a look at this now, but it occurs to me that Philip should at least have a screenshot or something to verify the changes, because he's the one that specified them.

File: dspace/config/news-xmlui.xml [hunk 1]
```diff
@@ -3,13 +3,13 @@
...
 
 <p>Want to publish a standalone dataset? Visit <xref target="https://data.lib.vt.edu/" rend="a-bold">VTechData</xref>.</p>
 
-<p>Faculty can deposit items to VTechWorks from the Electronic Faculty Activity Reporting System (Elements). Visit the Provost's <xref target="http://provost.vt.edu/faculty_affairs/efars.html" rend="a-bold">Elements page</xref> to learn more and to log in to Elements.</p>
+<p>Faculty can deposit items to VTechWorks from ELEments (EFARs). Visit the Provost's <xref target="http://provost.vt.edu/faculty_affairs/efars.html" rend="a-bold">Elements page</xref> to learn more and to log in to Elements.</p>
```
Comment (keithgee) [inline dspace/config/news-xmlui.xml]: @kdweeks - the capitalization of ELEments here might be a

Comment (keithgee) [reply dspace/config/news-xmlui.xml]: Sorry..ELEments might be a typo.

**Review by keithgee — CHANGES_REQUESTED** [review]
Kimberli - this file looks good, except for a possible typo with the capitalization of ELEments.

Would you be able to fix it, and then if everything else looks okay we can merge the branch? When I do the merge I might use the option that says to squash the commits - I noticed some of the commits in this pull request are test commits.

Thanks!

**Comment by kdweeks** [comment]
Oops yes, I'll fix that and send attach a screenshot.  Thanks so much!

**Comment by kdweeks** [comment]
@keithgee I fixed the typo.  @pyc1 I am attaching a screenshot for your review.
vtechworks_homepage

**Comment by pyc1** [comment]
Looks good Kimberli, thanks!

**Comment by kdweeks** [comment]
@pyc1 Thanks!

**Review by keithgee — APPROVED** [review]

**Comment by alawvt** [comment]
Looks good. @kdweeks, thank you for preparing this pull request and @keithgee and @pyc1 , thank you for reviewing and merging it.

**Pull request merged**